tools/csv-merger/csv-merger.go

```diff
@@ -935,6 +1056,10 @@ func main() {
 		log.Fatal("--crds-directory is required")
 	} else if *outputDir == "" {
 		log.Fatal("--olm-bundle-directory is required")
+	} else if *uxBackendServerImage == "" {
+		log.Fatal("--olm-bundle-directory is required")
+	} else if *uxBackendOauthImage == "" {
```

Comment:
if default is set, I don't think this value will ever be not present, better remove default as well and just have a comment here saying what image can be used for easy visilbility